media-plugins/mythplugins/metadata.xml

```diff
@@ -17,7 +17,7 @@
 		<flag name="raw">Allows MythGallery to view photos saved in RAW format</flag>
 		<flag name="mytharchive">Allows you to archive recorded content to CD or DVD</flag>
 		<flag name="mythbrowser">Adds a web browser within MythTV</flag>
-		<flag name="mythgallery">Adds a picture viewing gallery within MythTV</flag>
+		<flag name="mythgallery">Adds a picture viewing gallery within MythTV (removed in 31.0)</flag>
```

Comment:
This additional comment is superfluous, it's not a correct place to write down changelog into.

Reply:
Comment removed